src/_nebari/upgrade.py

```diff
@@ -364,7 +377,11 @@ def set_nested_item(config: dict, config_path: list, value: str):
             config[config_path[-1]] = value
 
         def update_image_tag(
-            config: dict, config_path: str, current_image: str, new_version: str
+            config: dict,
+            config_path: str,
+            current_image: str,
+            new_version: str,
+            attempt_fixes: bool = attempt_fixes,
```

Comment:
If `attempt_fixes` is exposed in `kwargs`, you can use it from there. I wouldn't even use a default value, but rather pass it explicitly in all the further calls to this function by grabbing it from `kwargs`

Reply:
Fixed